modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ReviewServiceTemplateRequest.java

```diff
@@ -22,6 +22,7 @@ public class ReviewServiceTemplateRequest {
     @Schema(description = "The result of review registration.")
     private ServiceReviewResult reviewResult;
 
+    @NotNull
     @Schema(description = "The comment of review registration.")
```

Comment:
Just use @NotNull and @NotBlank, the new exception `ReviewCommentValueMissingException` is unnecessary.

Reply:
updated.